service/src/main/java/org/cbioportal/service/impl/StructuralVariantServiceImpl.java

```diff
@@ -36,12 +40,42 @@
 
     @Autowired
     private StructuralVariantRepository structuralVariantRepository;
-    
+    @Autowired
+    private MutationRepository mutationRepository;
+    @Autowired
+    private MutationMapperUtils mutationMapperUtils;
+    @Autowired
+    private AlterationEnrichmentUtil<StructuralVariantCountByGene> alterationEnrichmentUtil;
+
     @Override
-    public List<StructuralVariant> fetchStructuralVariants(List<String> molecularProfileIds, 
-            List<Integer> entrezGeneIds,  List<String> sampleIds) {
-        
-        return structuralVariantRepository.fetchStructuralVariants(molecularProfileIds, entrezGeneIds, sampleIds);
+    public List<StructuralVariant> fetchStructuralVariants(List<String> molecularProfileIds,
+            List<Integer> entrezGeneIds, List<String> sampleIds) {
+
+        List<StructuralVariant> structuralVariants = mutationMapperUtils.mapFusionsToStructuralVariants(
```

Comment:
Nice.  Should we add a "TODO" comment here to remove once fusions are migrated to sv table in database?